.cirrus.yml

```diff
@@ -0,0 +1,35 @@
+env:
+  CARGO_HOME: /tmp/cargo
+
+task:
+  matrix:
+    - install_cmake_script: apt-get update && apt-get install -y cmake
+      matrix:
+        - name: test (linux)
+          container:
+            image: rust:latest
```

Comment:
Would we prefer rust:latest here or a specific pin? I'm actually open to it being latest here but a fixed in in Travis, but it should be a conscious decision.

Reply:
we could optionally limit this to the windows build. I don't have strong feelings here, but I miiiight lean towards keeping it pinned; if travis is down or we want to just switch at some point it would be nice to make sure we're kept honest.